guides/common/modules/con_managing-red-hat-subscriptions.adoc

```diff
@@ -4,37 +4,64 @@
 {ProjectName} can import content from the Red{nbsp}Hat Content Delivery Network (CDN).
 {Project} requires a Red{nbsp}Hat subscription manifest to find, access, and download content from the corresponding repositories.
 You must have a Red{nbsp}Hat subscription manifest containing a subscription allocation for each organization on {ProjectServer}.
-All subscription information is available in your Red Hat Customer Portal account.
+All subscription information is available on `console.redhat.com`.
 
 Use this chapter to import a Red{nbsp}Hat subscription manifest and manage the manifest within the {ProjectWebUI}.
 
 .Subscription allocations and organizations
-
 You can manage more than one organization if you have more than one subscription allocation.
 {Project} requires a single allocation for each organization configured in {ProjectServer}.
-The advantage of this is that each organization maintains separate subscriptions so that you can support multiple organizations, each with their own Red Hat accounts.
+The advantage of this is that each organization maintains separate subscriptions so that you can support multiple organizations, each with their own Red{nbsp}Hat accounts.
 
 .Future-dated subscriptions
-
 You can use future-dated subscriptions in a subscription manifest.
 When you add future-dated subscriptions to your manifest before the expiry date of the existing subscriptions, you can have uninterrupted access to repositories.
 
+.Subscriptions service capabilities
+The Subscriptions service is a service that helps organizations track and manage their Red{nbsp}Hat subscription usage across connected and disconnected environments. 
+* Allows customers to view subscription usage.
+* When configured, it displays usage data from both connected and disconnected environments.
+* Tracks Red{nbsp}Hat product usage and capacity at the fleet or account level.
```

Comment:
I'm unsure about what "fleet" means in this context. I'm aware that in the Cloud Console, there's a RH account and an organization to which accounts can belong. Does "fleet level" mean "organization level"?

Reply:
Yes I will change to **organization** level.